netci.groovy

```diff
@@ -69,7 +69,9 @@ class Constants {
     // the values are the environment variables
     def static jitStressModeScenarios = [
                'minopts'                        : ['COMPlus_TieredCompilation' : '0', 'COMPlus_JITMinOpts' : '1'],
-               'tieredcompilation'              : ['COMPlus_TieredCompilation' : '1'],
+               'tieredcompilation'              : ['COMPlus_TieredCompilation' : '1'], // this can be removed once tiered compilation is on by default
+               'no_tiered_compilation'          : ['COMPlus_TieredCompilation' : '0'],
+               'no_tiered_compilation_pri0'     : ['COMPlus_TieredCompilation' : '0'],
```

Comment:
IMO, this should be named `no_tiered_compilation_innerloop'. It should continue to be true that any job named "innerloop" runs pri-0 test only, and is automatically PR-triggered.

The idea is that if you ever manually trigger something (or it is push/cron triggered), we force pri-1 tests to run.

Reply:
👍